### ISSUE TITLE
New releases?

### ISSUE THREAD

**Issue by joyeecheung** [nodejs/llnode#161]
Refs: https://github.com/nodejs/admin/issues/34

Hi, I think it's time for a new release now (v1.6.3?). @hhellyer @rnchamberlain @indutny. I opened https://github.com/nodejs/admin/issues/34 to ask about the npm release process of foundation projects and it looks like we should open an issue in the project repo, so here it is. I can also do it if you agree to add me to the npm collaborators list (I assume the release process is just opening a PR like https://github.com/nodejs/llnode/pull/135 and then `npm publish` when it gets merged?).

**Comment by gibfahn** [comment]
FWIW I _think_ @hhellyer is out till Thursday, so this might have to wait till the end of the week.

**Comment by rnchamberlain** [comment]
I'm happy to add @joyeecheung to collaborators on https://www.npmjs.com/package/llnode if no-one objects. I think the above is all the release process amounts to, but yes, you might want to wait and check with @hhellyer.

**Comment by rnchamberlain** [comment]
No objections, I have added @joyeecheung as a collaborator on https://www.npmjs.com/package/llnode

**Comment by hhellyer** [comment]
@joyeecheung - Yes, I try to let the npm commands take care of things as much as possible. It's not a formal process and I'm not an npm expert either so feel free to make amendments or corrections.

From my own notes my steps are usually:
- Update the package.json via `npm version patch` and commit the change via a PR.
- Create a release in github to tag the code with the new version. (I usually put links to the main PR's in the release notes.)
- Do a test `npm install nodejs/llnode#vX.X.X` from github to check the new level does install ok.
- Using a clean clone (not the one I use for development in case of stuff lying around) `git checkout vX.X.X` and do `npm publish` from there.

**Comment by mmarchini** [comment]
@hhellyer now that Travis is working again, we could setup it to deploy every time a tag is created: https://docs.travis-ci.com/user/deployment/npm/#What-to-release. That way only steps one and two are required :)

**Comment by joyeecheung** [comment]
I just published v1.6.3, here is what I did

1. On the release branch, run `npm pack`, move `llnode-1.6.3.tgz` to a different location, `tar zxvf llnode-1.6.3.tgz` -> `cd packages` -> `npm install` to make sure it installs OK and does not contain redundant files (also this is how I discovered https://github.com/nodejs/llnode/pull/164)
1. Opened https://github.com/nodejs/llnode/pull/163 (this was actually done before 1), merged the commit using metadata generated with `get-metadata --repo llnode --readme ../node/README.md 163` (`get-metadata` comes from https://github.com/nodejs/node-core-utils)
2. `git pull` on the local master branch to bring in the release commit, `git tag v1.6.3`, and `git push upstream --tags`
3. In a different directory, `git clone git@github.com:nodejs/llnode.git` -> `cd llnode` -> `npm publish`. Although I believe since I had checked the tarball of `npm pack`, simply `npm publish` from the project directory should be fine, but a little bit extra care doesn't hurt.
